docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3'
+services:
+  website:
+    image: openchat 
+    build: .
+    ports:
+      - 8888:80
+    env_file:
+      - .env.docker
+    depends_on:
+      - mysql
+  mysql:
+    image: mysql/mysql-server
+    command: --default-authentication-plugin=mysql_native_password
+    environment:
+      - MYSQL_ROOT_PASSWORD=root_pass
```

Comment:
What's the use of .env file if we are exposing them in docker-compose itself.

Reply:
I will move them to the .env.docker file then

Reply:
As I see it .env files are great for situations where controlling the real environment variables is awkward, in the case of using docker-compose, or deploying to Kubernetes, using environment variables is usually the better and easier approach.
While env fils are well suited to local development.
What do you think @ankitjain28may?

Reply:
I just read the documentation of `env_file` in docker-compose, and wow, I did not realise it supported .env files directly :-) that is going to make things nicer sometimes!

Reply:
yeah, it supports `.env`